pkg/symbol/symbol.go

```diff
@@ -65,7 +65,7 @@ func (s *Symbolizer) Symbolize(prof *profiler.Profile) error {
 	userJITedFunctions, err := s.resolveJITedFunctions(pid, prof.UserLocations)
 	if err != nil {
 		// We expect only a minority of processes to have a JIT and produce the perf map.
-		if !errors.Is(err, perf.ErrNotFound) {
+		if !errors.Is(err, perf.ErrPerfMapNotFound) {
```

Comment:
I think the negation here is a mistake
```suggestion
		if errors.Is(err, perf.ErrPerfMapNotFound) {
```

Reply:
Good catch !